src/components/button/templates/component/style.js

```diff
@@ -100,81 +109,96 @@ export let componentStyle = `
         visibility: hidden;
     }
 
-    .paypal-button .paypal-button-content::before {
-        content: "";
-        position: absolute;
-        z-index: -1;
-        width: 100%;
-        height: 100%;
-    }
-
-    .paypal-button .paypal-button-tag-content {
+    .paypal-button .paypal-tagline {
         max-width: 100%;
         font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
         font-weight: normal;
-
         display: block;
         text-align: center;
         width: auto;
-        font-size: 9px;
         margin-top: 2px;
         visibility: hidden;
     }
 
-    .paypal-button .paypal-button-tag-content.paypal-tagline-color-blue {
+    .paypal-button .paypal-tagline.paypal-tagline-color-${ TAGLINE_COLOR.BLUE } {
         color: #003366;
     }
 
-    .paypal-button .paypal-button-tag-content.paypal-tagline-color-black {
-        color: #000;
+    .paypal-button .paypal-tagline.paypal-tagline-color-${ TAGLINE_COLOR.BLACK } {
+        color: #6C7378;
```

Comment:
Is this color a PayPal brand color? Looks a little lighter than the current tagline for the black button.